homework-2/public/index.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Sofa\Homework;
+
+use PDO;
+
+include __DIR__.'/../src/Autoloader.php';
+include __DIR__.'/../bin/db.php';
+
+$dsn = 'pgsql:host=localhost;dbname=postgres';
+$conn = new PDO($dsn.';user=sofa;password=sofa');
+
+?>
+
+<div>
+    <style>
+        table {
+            border-collapse: collapse;
+        }
+        th, td {
+            border: 1px solid black;
+            padding: 5px;
+        }
+    </style>
+    <table>
+        <tbody>
+        <?php
+            if (empty($_GET)){
+                $sports = selectAllSports ($conn);
+                foreach ($sports as $sport)
+                {
+                    ?>
+                    <tr>
+                        <th colspan="2">List of sports</th>
+                    </tr>
+                    <tr>
+                        <td><?= $sport['name'] ?></td>
+                        <td><a href="/?page=sport&amp;slug=<?php echo $sport["slug"] ?>">Details</a></td>
+                    </tr>
+                    <?php
+                }
+            }
+
+            if (isset($_GET["page"]) and $_GET["page"]=="sport")
```

Comment:
Ovdje si moga elseif koristiti, jer ako je prvi if zadovoljen, ne treba dalje gledati
```suggestion
            elseif (isset($_GET["page"]) and $_GET["page"]=="sport")
```